### ISSUE TITLE
Is loss 4.x?

### ISSUE THREAD

**Issue by luuuyi** [wy1iu/sphereface#48]
Hi, actually, follow your instruction, after 28000 iterations, the loss decrease to 4.x. So, my question is one model can be used correct, what loss should be decreased to? Or, I should reset the max iterations, 10000 or 200000?
Thank you

**Comment by nyyznyyz1991** [comment]
for 20 layers, my loss value is 1.x and lfw test accuracy is 99.22%

**Comment by luuuyi** [comment]
how many iterations do you train? Can I have a look on your solver.prototxt?@nyyznyyz1991

**Comment by nyyznyyz1991** [comment]
i followed the author's solver,prototxt, didn't change anything. 
28000 iters

base_lr: 0.1
display: 100
max_iter: 28000
lr_policy: "multistep"
gamma: 0.1
momentum: 0.9
weight_decay: 0.0005
snapshot: 1000
snapshot_prefix: "result/sphereface_model"
solver_mode: GPU
device_id: 0
net: "code/sphereface_model.prototxt"
train_state {
  level: 0
  stage: ""
}
stepvalue: 16000
stepvalue: 24000
stepvalue: 28000

**Comment by luuuyi** [comment]
@nyyznyyz1991 ok, i see it. Some more question, have you follow the author's instruction one by one? Using the Matlab script to align the face, then train them? I ask this question because I just crop the face without aligning the face(using the five points). May be my problem happens at this?

**Comment by xhzcyc** [comment]
My LFW training loss is about 0.01

**Comment by Allloginsbusy** [comment]
@luuuyi, I have the same results. I have it with align and without. Are you solve this problem?
